src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -323,16 +323,16 @@ public void GetFolderPath_Unix_SpecialFolderDoesNotExist_CreatesSuccessfully(Env
         [Fact]
         public void GetSystemDirectory()
         {
-            if (PlatformDetection.IsWindowsNanoServer)
-            {
-                // https://github.com/dotnet/corefx/issues/19110
-                // On Windows Nano, ShGetKnownFolderPath currently doesn't give
-                // the correct result for SystemDirectory.
-                // Assert that it's wrong, so that if it's fixed, we don't forget to
-                // enable this test for Nano.
-                Assert.NotEqual(Environment.GetFolderPath(Environment.SpecialFolder.System), Environment.SystemDirectory);
-                return;
-            }
+            //if (PlatformDetection.IsWindowsNanoServer)
```

Comment:
Why not delete it instead of commenting it out?

Reply:
Ops this was included by mistake is not supposed to be part of this PR.